docs/_docs/auth.md

```diff
@@ -69,25 +71,33 @@ _→ [Add Digest Auth to your Zapier Integration](https://platform.zapier.com/do
+The _Connection Label_ field in your app's authentication options lets you customize the account connections for your integration to include a username, email address, name, or other identifiable information in the connection label. That helps users distinguish between their authenticated accounts.
 
 ![Zapier Authentication Connection Label](https://cdn.zapier.com/storage/photos/f2c3d557023ce2a65b41122da34c1fdd.png)
 
-Customize your app's connection label from the _Connection Label_ field in your Zapier integration. Zapier always includes the app's name in each account label. You can additionally include:
+Zapier always includes the app's name in each account label. You can add:
 
 - Plain text that will be included after your app's full name
-- Input fields with data users entered in the authentication form, referenced with {% raw %}`{{bundle.authData.field}}`{% endraw %} variables, replacing `field` for your input field name. With Basic auth, use `username` as the field.
-- Output fields from your app's authentication test API call, referenced with {% raw %}`{{bundle.inputData.field}}`{% endraw %} variables, replacing `field` for your API output field name.
+- Data that users entered in the authentication form.
+- Output fields from your app's authentication test API call.
+
+To add a field value to the connection label, enter the name of the field, surrounded by double curly braces, in the _Connection Label_ field.
+
+For example, if your auth configuration includes a `username` field, your connection label could look like:
+
+{% raw %}`{{username}}`{% endraw %}
 
-To add a connection label using an input field, enter the following in your _Connection Label_ field, optionally replacing `username` with the key for another input field your authentication form includes:
+Alternately, you can use an output field from your test API call in the Connection Label. If you aren't sure what fields will be returned, skip this step for now and go to Step 3, _Test your Authentication_.
 
-{% raw %}`{{bundle.authData.username}}`{% endraw %}
+After connecting an account, check the _Response_ tab to see the fields returned from the test request. Then, return to Step 2, and enter that field key in the _Connection Label_ field. So if the field you want to use is called `team`, enter:
 
-Alternately, you can use an output field from your test API call in the Connection Label if you know the name of the field you wish to use that your API sends to Zapier. Otherwise, skip adding the connection label first, and instead click _Connect an Account_ under step 2. Then click _Test Connected Account_ and check the _Response_ tab for a field that includes the data your app should use in the connection label. Finally, enter that field key in the following text instead of `field`, and add it to your _Connection Label_ field:
+{% raw %}`{{team}}`{% endraw %}
```

Comment:
If `team` is not an authentication input field or stored in a connected account, `{{team}}` won't work. It still needs to be:

```suggestion
{% raw %}`{{bundle.inputData.team}}`{% endraw %}
```

The rule of thumb can be: Try `{{something}}` (equivalent to `{{bundle.authData.something}}`) first. If `{{something}}` doesn't work, use `{{bundle.inputData.something}}`.

Reply:
@eliangcs I did a lot of testing on this and requested early feedback on that via Slack, but I didn't get a lot of feedback from people that way. 

After the testing I've done in, I'm confident that the hoisting occurs for both authData and inputData - that's exactly why I proposed these changes. If you have contrary evidence from testing, could you share that? Otherwise, I'd like to go ahead with the substance of the changes as proposed.

Reply:
@armchairlinguist I did some tests, and you're right! Both authData and inputData are hoisted. I'm really sorry for the confusion! 🙇 

Feel free to go ahead and use your original text.